libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -41,16 +40,16 @@ export interface RenderComponentOptions<TContext extends object> {
   readonly injector: Injector;
 }
 
-export class RenderOverride<TContext extends object> {
-  constructor(public renderer: IRenderFunction<TContext>) { }
+export interface DefaultRenderOptions<TContext extends object> {
```

Comment:
Thinking out load here - the `DefaultRenderOptions` doesn't convey the message that it allows mutating the props. If you have suggestions for a better/other name - I'm open for suggestions, but if not - please add a JDoc comment explaining what the purpose of this object is.